code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -21,8 +20,6 @@
 		hide_healing = TRUE

```

Comment:
you should probably remove this since, well, these are all much more limited symptoms